collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploaderTask.java

```diff
@@ -211,21 +211,17 @@ private boolean uploadOneSubmission(String urlString, String id, String instance
             } catch (ClientProtocolException | ConnectTimeoutException | UnknownHostException | SocketTimeoutException | HttpHostConnectException e) {
                 if (e instanceof  ClientProtocolException) {
                     outcome.mResults.put(id, fail + "Client Protocol Exception");
-                    Timber.e(e, "Client Protocol Exception thrown due to : %s ", e.getMessage());
```

Comment:
Please put this back in. I understand it's not necessary, but I'd rather have all those in a single PR.